flask_resty/view.py

```diff
@@ -183,11 +192,7 @@ def make_created_response(self, item):
         :return: The HTTP response
         :rtype: :py:class:`flask.Response`
         """
-        response = self.make_item_response(item, 201)
-        location = self.get_location(item)
-        if location is not None:
-            response.headers["Location"] = location
-        return response
+        return self.make_item_response(item, 201)
```

Comment:
you want to amend the doc of this method

Reply:
https://github.com/4Catalyzer/flask-resty/pull/328/files#diff-094ac497a5d94f5561a8a35fa2496623R101 does it, no?

Reply:
@itajaja does that answer the question, or am i missing something here?

Reply:
yeah of course 🤦‍♂ . all good